public/js/clients/firefox.js

```diff
@@ -102,7 +102,6 @@ function initPage(actions) {
 
   setupCommands({ threadClient, tabTarget, debuggerClient });
 
-  tabTarget.on("will-navigate", actions.willNavigate);
```

Comment:
while working on this patch i noticed that `will-navigate` fires twice for debuggee refreshes. This is because `frame-update` fires for a will navigate event when the page is paused. I removed this event because i believe it is redundant now.